### PR TITLE
Popup buffer: bigger size but not too much, get the focus, have markown syntax highlithg

### LINKED ISSUE #1: Small rapid improvements: filtype, focus and dimension of the result popup
The result popup would be better if 

- it was a little bigger. 60 x 22 is too small
- it has focus after displaying the results
- has the "markdown" filetype, because Gemini result is always Markdown formatted

### PR DESCRIPTION
**Pull request by rakotomandimby**
Fixes #1

### CODE REVIEW

**Comment by gera2ld** [comment]
I think `focus` should be optional and default as `false` when the popup is used to show some hints. Could you add an option to `M.opts`?

**Comment by rakotomandimby** [comment]
@gera2ld , OK I will do that.

**Comment by rakotomandimby** [comment]
@gera2ld , feel free to review.
I also updated the README in order to document that parameter.

File: README.md [hunk 1]
```diff
@@ -21,6 +21,10 @@ Using lazy.nvim:
     -- The locale for the content in the locale above to be translated into
     alternate_locale = 'zh',
     -- Define custom prompts here, see below for more details
+    result_popup_get_focus = false,
+    -- Gemini's answer is displayed in a popup buffer
+    -- Default behaviour is not to give it the focus because it is seen as a kind of tooltip
+    -- But if you prefer it to get the focus, set to true.
     prompts = {},
```
Comment (gera2ld) [inline README.md]: Seems wrong location of comments, could you swap the comments? Otherwise it looks good to me.

**Comment by rakotomandimby** [comment]
@gera2ld I fixed it.

**Comment by gera2ld** [comment]
Thank you so much.

**Pull request merged**